sites/fast-website/src/app/components/feature-card/feature-card.styles.ts

```diff
@@ -0,0 +1,69 @@
+import { css } from "@microsoft/fast-element";
+import { display, neutralForegroundRestBehavior } from "@microsoft/fast-components";
+
+export const FeatureCardStyles = css`
+    ${display("grid")} :host {
+        contain: layout;
+        grid-template-columns: 1fr 2fr;
+        color: var(--neutral-foreground-rest);
+        font-family: var(--text-font);
+        box-sizing: border-box;
+        padding: 20px;
+        box-shadow: unset;
+        height: 140px;
+        width: 600px;
```

Comment:
What happens if my screen size is 320px?

Reply:
If I remove the width and scale it down, things get pretty tight.  It would look better at that point to reorient the card so the number and header are above the other content which I am happy to do.

Reply:
Added responsive behavior to the card.